apps/root/src/pages/earn/components/strategies-table/components/columns/index.tsx

```diff
@@ -235,17 +250,20 @@ export const portfolioColumnConfigs: StrategyColumnConfig<StrategiesTableVariant
         </StyledBodySmallRegularTypo2>
       </ContainerBox>
     ),
+    hidden: (theme) => window.innerWidth < theme.breakpoints.values.lg,
```

Comment:
I dont like that we use this repetitively, could we do it like the Hidden component and just send hidden from which breakpoint it should be and handle it on the table?